src/isar/telemetry/telemetry.py

```diff
@@ -0,0 +1,87 @@
+import logging
+
+import backoff
+from paho.mqtt import client as mqtt
+from paho.mqtt.client import Client
+
+from isar.config.settings import settings
+from robot_interface.models.telemetry.entity import Entity
+
+
+class Telemetry:
+    """This is a wrapper for the mqtt client."""
+
+    def __init__(self) -> None:
+        self.logger = logging.getLogger("mqtt_client")
```

Comment:
```suggestion
        self.logger = logging.getLogger("telemetry")
```
Rename all places the `mqtt_client` logger is used to `telemetry`. It's a better name.